main/catch2/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "catch2"
+pkgver = "3.5.2"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCATCH_BUILD_EXTRA_TESTS=ON",
+    "-DCATCH_DEVELOPMENT_BUILD=ON",
```

Comment:
```suggestion
    "-DCATCH_BUILD_TESTING=ON",
```

the other stuff doesn't have to be enabled